releases.cfg

```diff
@@ -17,6 +17,7 @@ Releases
   {
     Modules = DIRAC, Web, VMDIRAC
     Externals = v6r6p3
+    DIRACOS = master
```

Comment:
Is it DIRACOS or DiracOS ?

Reply:
DIRACOS

Reply:
def getDiracOSVersion(self, diracOSVersion=None):
    """
    It returns the DIRACOS version
    :param str diracOSVersion: the OS version
    """

    if diracOSVersion:
      return diracOSVersion
    try:
      return self.prjRelCFG[self.projectName][cliParams.release].get(
          "Releases/%s/**DiracOS**" % cliParams.release, diracOSVersion)
    except KeyError:
      pass
return diracOSVersion

Reply:
@zmathe you told me to use `DIRACOS`. So what is correct ?

Reply:
It must be DIRACOS like: https://github.com/DIRACGrid/DIRAC/blob/integration/Core/scripts/dirac-install.py#L746

It is a bug... going to fix it.

Reply:
@chaen @atsareg Fixed in #3957